Source/JavaScriptCore/heap/HeapUtil.h

```diff
@@ -46,7 +46,7 @@ class HeapUtil {
     // before liveness data is cleared to be accurate.
     template<typename Func>
     static void findGCObjectPointersForMarking(
-        Heap& heap, HeapVersion markingVersion, HeapVersion newlyAllocatedVersion, TinyBloomFilter filter,
+        Heap& heap, HeapVersion markingVersion, HeapVersion newlyAllocatedVersion, TinyBloomFilter<> filter,
```

Comment:
Maybe this should have an explicit template argument to make sure we don't accidentally convert types down below

Reply:
Updated